contexts/v1.jsonld

```diff
@@ -2,7 +2,7 @@
   "@context": {
     "@protected": true,
```

Comment:
I wonder if just include an `@vocab` would be a better move here.

Reply:
Why do you think that, @OR13? If we do that, then everything under StatusList2021 becomes a part of the status list vocabulary. Seems like a dangerous thing to just blanket map any term to the status list vocabulary (this is why `@vocab`, in general, is a bad idea and shouldn't be used).